pom.xml

```diff
@@ -256,6 +256,87 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>parallelTest</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <version>${maven.surefire.plugin.version}</version>
+                        <executions>
+                            <execution>
+                                <id>default-test</id>
+                                <phase>test</phase>
+                                <goals>
+                                    <goal>test</goal>
+                                </goals>
+                                <configuration combine.self="override">
+                                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                                    <!-- 1C means 1 process per cpu core -->
+                                    <forkCount>4C</forkCount>
+                                    <reuseForks>true</reuseForks>
+
+                                    <!-- the argLine variable is needed for jacoco-->
+                                    <argLine>
+                                        -Xms128m -Xmx1G -XX:MaxPermSize=128M
+                                        -Dhazelcast.version.check.enabled=false
+                                        -Dhazelcast.mancenter.enabled=false
+                                        -Dhazelcast.logging.type=none
+                                        -Dhazelcast.test.use.network=false
+                                        -Dhazelcast.test.multiple.jvm=true
+                                    </argLine>
+                                    <includes>
+                                        <include>**/**.java</include>
+                                    </includes>
+                                    <excludes>
+                                        <exclude>**/jsr/**.java</exclude>
+                                    </excludes>
+                                    <groups>com.hazelcast.test.annotation.ParallelTest AND com.hazelcast.test.annotation.QuickTest</groups>
```

Comment:
neither me, afaik it's not documented. I found the `AndGroupMatcher` while trying to fix another issue.